keps/sig-apps/4368-support-managed-by-label-for-batch-jobs/README.md

```diff
@@ -0,0 +1,1158 @@
+  - [Alternative ideas to improve debuggability](#alternative-ideas-to-improve-debuggability)
+    - [Condition to indicate Job is skipped](#condition-to-indicate-job-is-skipped)
+    - [Event indicating the Job is skipped](#event-indicating-the-job-is-skipped)
+- [Infrastructure Needed (Optional)](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+<!--
+**ACTION REQUIRED:** In order to merge code into a release, there must be an
+issue in [kubernetes/enhancements] referencing this KEP and targeting a release
+milestone **before the [Enhancement Freeze](https://git.k8s.io/sig-release/releases)
+of the targeted release**.
+
+For enhancements that make changes to code or processes/procedures in core
+Kubernetes—i.e., [kubernetes/kubernetes], we require the following Release
+Signoff checklist to be completed.
+
+Check these off as they are completed for the Release Team to track. These
+checklist items _must_ be updated for the enhancement to be released.
+-->
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [x] (R) Design details are appropriately documented
+- [x] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [x] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md)
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [x] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+We support the `managed-by` label as a lightweight mechanism to delegate the Job
+synchronization to an external controller.
```

Comment:
Official labels are public, so it'd be `job.kubernetes.io/managed-by` or similar. Don't use a private label (one with no `/` character separating a DNS domain from the rest of the key).

Reply:
Sure, makes sense, the full name of the proposed label is `batch.kubernetes.io/managed-by`. And it was mentioned in the Design details. However, I admit it could be confusing as troughout the KEP the name is `managed-by` in skewed quotes suggesting this is the technical name. 

To minimize the changes of confusion of other readers:
- I mention early in the proposal that the full name is `batch.kubernetes.io/managed-by`
- replace the skew quotes with " to make it less technical when used in various places of the text

Still, I think it is ok to call it just "managed-by" in most places of the text, because otherwise it looks verbose, and "managed-by" reads better. PTAL.